src/Compilation/Config.zig

```diff
@@ -64,6 +64,8 @@ san_cov_trace_pc_guard: bool,
 
 pub const CFrontend = enum { clang, aro };
 
+pub const LtoMode = enum { none, flto, auto, full, thin, jobserver };
```

Comment:
```suggestion
pub const LtoMode = enum { none, thin, full };
```

https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-flto

Clang docs say that `-flto`, `-flto=full`, `-flto=auto`, and `-flto=jobserver` are all equivalent. So let's just simplify this.